<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataDialog</class>
 <widget class="QDialog" name="dataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Move Kodi Data</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listDirectories1">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>70</y>
     <width>180</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>100</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to select an external partition &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    border-color: black;
  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>External storage</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dataButton3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>371</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Move thumbnails from /sdcard to selection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose operation below</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dataButton1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>203</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Move data from /sdcard/ to selection</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="dataButton4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Move thumbnails from selection to /sdcard</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dataButton2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>222</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Move data from selection to /sdcard/</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>dataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
